rest/entityId.js

```diff
@@ -96,12 +99,7 @@ class EntityId {
     // shard, realm, and num take 4, 8, and 8 bytes respectively from the left
     return this.num === null
       ? null
-      : [
-          constants.HEX_PREFIX,
-          toHex(this.shard).padStart(8, '0'),
-          toHex(this.realm).padStart(16, '0'),
-          toHex(this.num).padStart(16, '0'),
-        ].join('');
+      : `${constants.HEX_PREFIX}${longFormEvmAddressPrefix}${toHexString(this.num, false, 16)}`;
```

Comment:
Why concatenate three strings when should be able to use:

```suggestion
      : toHexString(this.num, true, 40);
```

Reply:
choice between readability and slightly better performance.